pkg/plugin/sdk/v1alpha2/plugin/grpc.go

```diff
@@ -147,6 +188,41 @@ func (m *GRPCClient) PostBuildHook(isInteractiveMode bool, promptRunner ioutils.
 	return err
 }
 
+// CustomCommands is called from the Core to execute the Plugin CustomCommands.
+// It returns a list of commands that the plugin implements.
+func (m *GRPCClient) CustomCommands() ([]*Command, error) {
+	req := &proto.CustomCommandsReq{}
+	customCommandsPB, err := m.client.CustomCommands(context.Background(), req)
+
+	customCommands := make([]*Command, 0)
+
+	for _, pbCommand := range customCommandsPB.Commands {
+		customCommands = append(customCommands, &Command{
+			Use:       pbCommand.Use,
+			ShortDesc: pbCommand.ShortDesc,
+			LongDesc:  pbCommand.LongDesc,
+			Run:       nil,
```

Comment:
Why is the implementation `nil` in this case? Is this just to transfer the metadata about it?

Reply:
Should the metadata vs implementation be split up? 🤔

Reply:
The run field can actually just be removed.